toolkit/authentication/src/androidTest/java/com/arcgismaps/toolkit/authentication/TestAuthenticationUtils.kt

```diff
@@ -0,0 +1,77 @@
+package com.arcgismaps.toolkit.authentication
+
+import android.content.Intent
+import android.net.Uri
+import com.arcgismaps.httpcore.ArcGISHttpClient
+import com.arcgismaps.httpcore.Request
+import com.arcgismaps.httpcore.Response
+import com.arcgismaps.httpcore.authentication.ArcGISAuthenticationChallenge
+import io.mockk.every
+import io.mockk.mockk
+
+/**
+ * A mock [ArcGISAuthenticationChallenge] for testing purposes.
+ *
+ * @since 200.5.0
+ */
+fun makeMockArcGISAuthenticationChallenge() = mockk<ArcGISAuthenticationChallenge>().apply {
+    every { requestUrl } returns "https://arcgis.com"
+    every { cause } returns Throwable()
+}
+
+/**
+ * Returns an intent that simulates a successful redirect from the OAuth server.
+ *
+ * @since 200.5.0
+ */
+fun getSuccessfulRedirectIntent(redirectUrl: String): Intent {
```

Comment:
nitpick naming suggestion:
```suggestion
fun createSuccessfulRedirectIntent(redirectUrl: String): Intent {
```